lib/triggers/queryOrders.js

```diff
@@ -33,20 +33,37 @@ function getMetaModel(cfg, callback) {
 }
 
 function processTrigger(msg, cfg, next, snapshot) {
-    var self = this;
 
+    var self = this;
     var client = sphere.createClient(cfg);
-
     var lastModifiedAt = snapshot.lastModifiedAt || '1970-01-01T00:00:00.000Z';
 
-    var where = 'lastModifiedAt > "' + lastModifiedAt + '"';
+    var snapshot = snapshot || {};
+    snapshot.reboundedOrders = snapshot.reboundedOrders || {};
 
-    if (cfg.where) {
-        where += ' and ' + cfg.where;
+    function queryOrders() {
+
+        var where = 'lastModifiedAt > "' + lastModifiedAt + '"';
+        if (cfg.where) {
+            where += ' and ' + cfg.where;
+        }
+
+        return client.orders
+            .where(where)
+            .expand('syncInfo[*].channel')
+            .perPage(20)
+            .sort('lastModifiedAt', true)
+            .fetch();
     }
 
-    // returns ordersResponse
-    function queryOrders() {
+    function queryReboundedOrders(reboundedOrderIds) {
+
+        var where = 'id in ("' + reboundedOrderIds.join('","') + '")';
+
+        if (cfg.where) {
+            where += ' and ' + cfg.where;
```

Comment:
Do we really need to concatenate it ``cfg.where``? Don't we just want to get those whose id is stored in the snapshot?